package.json

```diff
@@ -91,9 +91,11 @@
     "multiselect": "~0.9.12",
     "number_helpers": "^0.1.1",
     "patternfly": "^3.58.0",
-    "patternfly-react": "^2.30.6",
+    "patternfly-react": "^2.33.0",
+    "patternfly-react-extensions": "^2.18.2",
     "prop-types": "^15.6.0",
     "react": "^16.8.1",
+    "react-ace": "^6.3.2",
```

Comment:
https://github.com/theforeman/foreman-packaging/pull/3706

Reply:
i still think you need to remove the brace from this file, it is no longer imported by foreman (rather via by react-ace)

Reply:
@ohadlevy  we need it in `EditorView` - https://github.com/theforeman/foreman/pull/6180/files#diff-43445d906144300b418f2f9a8c540820R5

as stated in Ace docs - https://github.com/securingsincity/react-ace/blob/master/docs/Ace.md

Reply:
does it make sense to export brace from react-ace? i think its wrong to require it twice (here and at brace) perhaps send them a pr to export it? then we wont have two copies of it...?

Reply:
@glekner https://github.com/securingsincity/react-ace/issues/638 ?

Reply:
ok, can you align the brace version again (so they dont conflict with react-ace?) i think this should be enough to unblock this PR. I already created a packaging PR at https://github.com/theforeman/foreman-packaging/pull/3706

Reply:
aligned